pkg/kube/roles.go

```diff
@@ -180,3 +187,79 @@ func userRolesFor(userKind string, userName string, envRoles map[string]*v1.Envi
 	}
 	return answer
 }
+
+// CreateClusterRoleBinding creates or extends a cluster role binding in a given namespace for a service account
+func CreateClusterRoleBinding(kubeClient kubernetes.Interface, namespace string, name string,
```

Comment:
This doesnt' actually do what the method/godoc implies (in that if it already exists, it just silently exists and does nothing).
I suggest either modifying the name/documentation or splitting into 2 methods, one to check if it exists and one to always Create it